src/applications/vaos/new-appointment/components/ContactInfoPage.jsx

```diff
@@ -91,21 +100,27 @@ const uiSchema = {
 const pageKey = 'contactInfo';
 const pageTitle = 'Your contact information';
 
-export function ContactInfoPage({
-  schema,
-  data,
-  pageChangeInProgress,
-  openFormPage,
-  updateFormData,
-  routeToNextAppointmentPage,
-  routeToPreviousAppointmentPage,
-}) {
-  const history = useHistory();
+export default function ContactInfoPage() {
   useEffect(() => {
-    openFormPage(pageKey, uiSchema, initialSchema);
     document.title = `${pageTitle} | Veterans Affairs`;
     scrollAndFocus();
   }, []);
+  const history = useHistory();
+  const dispatch = useDispatch();
```

Comment:
I also ended up removing `connect()` in here, since it was easier to use the React-Redux hooks.